packages/xrpl/test/integration/transactions/xchainClaim.test.ts

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable max-statements -- necessary because transfers require a lot of steps */
+import { assert } from 'chai'
+import { encode } from 'ripple-binary-codec'
+import { sign } from 'ripple-keypairs'
+
+import {
+  SignerListSet,
+  Wallet,
+  XChainAddClaimAttestation,
+  XChainBridge,
+  XChainClaim,
+  XChainCreateBridge,
+  XChainCreateClaimID,
+  xrpToDrops,
+} from '../../../src'
+import serverUrl from '../serverUrl'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import {
+  generateFundedWallet,
+  GENESIS_ACCOUNT,
+  getXRPBalance,
+  testTransaction,
+} from '../utils'
+
+// how long before each test case times out
+const TIMEOUT = 20000
+
+describe('XChainCreateBridge', function () {
+  let testContext: XrplIntegrationTestContext
+
+  beforeEach(async () => {
+    testContext = await setupClient(serverUrl)
+  })
+  afterEach(async () => teardownClient(testContext))
+
+  it(
+    'base',
+    async () => {
+      const signatureReward = '1000000'
+      const xchainBridge: XChainBridge = {
+        LockingChainDoor: testContext.wallet.classicAddress,
+        LockingChainIssue: { currency: 'XRP' },
+        IssuingChainDoor: GENESIS_ACCOUNT,
+        IssuingChainIssue: { currency: 'XRP' },
+      }
+      const setupTx: XChainCreateBridge = {
```

Comment:
Could this setup code be moved to a utility method?  Similar to what was done for the AMM integration tests.